subsys/net/l2/ieee802154/ieee802154.c

```diff
@@ -338,6 +338,11 @@ void ieee802154_init(struct net_if *iface)
 	ctx->channel = IEEE802154_NO_CHANNEL;
 	ctx->flags = NET_L2_MULTICAST;
 
+#if defined(CONFIG_IEEE802154_NET_IF_NO_AUTO_START)
```

Comment:
Please change this to `if (IS_ENABLED(CONFIG_IEEE802154_NET_IF_NO_AUTO_START)) {` as that is the preferred way.

Reply:
done